src/main/java/com/google/api/codegen/nodejs/NodeJSGapicContext.java

```diff
@@ -495,4 +578,16 @@ public String getByteLengthFunction(TypeRef typeRef) {
               "options",
               "callback")
           .build();
+
+  private static final ImmutableSet<String> COMMON_PROTO_PATHS =
```

Comment:
It doesn't seem clean to hardcode such information as this and `getFileUrl` into `NodeJSGapicContext`. Will there be a better solution under MVVM?